app/javascript/react/components/molecules/StationValueLabel/StationValueLabel.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import BroadCastLogo from "../../../assets/icons/broadCastLogo.svg";
+import { LargeH1, H3 } from "../../../assets/styles/Typography";
+import * as S from "./StationValueLabel.style";
+
+interface StationValues {
+  date: string;
+  value: number;
+  parameter: string;
+}
+
+const StationValueLabel = ({ date, value, parameter }: StationValues) => {
+  return (
+    <S.GradientBox>
+      <S.GradientImage>
+        <img src={BroadCastLogo} alt="Broadcasting logo" />
+      </S.GradientImage>
+      <S.TextContainer>
+        <H3>Avg for {date}</H3>
```

Comment:
Are we planning to have `Strings` file? If yes, why not start now :D

Reply:
We used to make a separate file for all hard-coded Strings, for future localizing 🤔

Reply:
Ok, thanks, I'll create the task for adding `i18n` to handle texts :)